### PR TITLE
feat: playground aspect-ratio configuration added

### PR DESCRIPTION
**Pull request by rxb3rth**

### CODE REVIEW

File: playground/pages/index.jsx [hunk 1]
```diff
@@ -637,6 +637,19 @@ const LiveSatori = withLive(function ({ live }) {
...
+              <button type='button' onClick={() => {
+                setWidth(1000)
+                setHeight(1000)
+              }}>1:1</button>
+              <button type='button' onClick={() => {
+                setWidth(1000)
+                setHeight((1000 * 9) / 16)
+              }}>16:9</button>
+            </div>
+          </div>
```
Comment (styfle) [inline playground/pages/index.jsx:652]: This seems to be the same as the existing sliders.

Can we avoid adding another row and instead put the buttons to the right of the slider?

I'm not even suer 1:1 is even that useful for OG Images.

We probably want 1200 x 600 (2:1) and 1200 x 630 (1.9:1) instead.

Comment (shuding) [reply playground/pages/index.jsx:652]: > Can we avoid adding another row and instead put the buttons to the right of the slider?

Agreed. Maybe we can change the `Reset` button to `Set to 2:1` and `Set to 1.9:1`.

Comment (rxb3rth) [reply playground/pages/index.jsx:652]: @styfle @shuding 

Maybe like this?

image

Comment (shuding) [reply playground/pages/index.jsx:652]: That looks good!

Comment (styfle) [reply playground/pages/index.jsx:652]: Looks like a typo. Should be `1.9:1`

**Comment by shuding** [comment]
FYI the deployment failed with the following errors:

```log


Failed to compile.
--
20:34:46.731 |  
20:34:46.731 | ./pages/index.jsx
20:34:46.731 | 388:6  Warning: React Hook useEffect has an unnecessary dependency: 'overrideOptions'. Either exclude it or remove the dependency array. Outer scope values like 'overrideOptions' aren't valid dependencies because mutating them doesn't re-render the component.  react-hooks/exhaustive-deps
20:34:46.732 | 405:12  Error: 'width' is already declared in the upper scope on line 373 column 10.  @typescript-eslint/no-shadow
20:34:46.732 | 405:19  Error: 'height' is already declared in the upper scope on line 374 column 10.  @typescript-eslint/no-shadow
20:34:46.732 | 833:6  Warning: React Hook useEffect has missing dependencies: 'activeCard' and 'onChange'. Either include them or remove the dependency array.  react-hooks/exhaustive-deps
20:34:46.732 |  
20:34:46.732 | info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
20:34:46.740 | ELIFECYCLE  Command failed with exit code 1.
20:34:46.770 | Error: Command "cd .. && pnpm build && cd playground && pnpm build" exited with 1


```

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**